src/components/Meditation.vue

```diff
@@ -42,5 +42,20 @@
 </script>
 
 <style lang="scss" scoped>
+  @import '../assets/sass/_variables';
 
-</style>
\ No newline at end of file
+  a {
+    color: $color-grey !important;
```

Comment:
Do we need !important?

Reply:
I'm not sure. the `scoped` attribute gives me assurance in this not being as 'deadly' as a project-wide `!important`. The reason for doing was because it's coming from the CMS so can't add its own class. The other option is...

```
#meditation {
    a { color: $color-grey !important; }
}
```

But the encapsulation is implicit because of `scope` and a global stylie handling the colour of links is higher in weight. Something to play with.